added pip -r

shyamgupta196 opened the pull request.

shyamgupta196 committed:
Update postBuild

shyamgupta196 committed:
Merge branch 'BDA-KTS:master' into master

shyamgupta196 committed:
adding pip -r

taimoorkhan-nlp commented:
@shyamgupta196 

I used the postbuild file defined in the guidelines here: https://github.com/GESIS-Methods-Hub/guidelines
Where did you get the updated code from?

shyamgupta196 commented:
Since this was open for review already, I just wanted to double check if everything was running smoothly or not, so I encountered this bug , where the cell wasn't installing requirements.txt . This PR specifically addressed that issue. 

I think the postbuild you are using is an older postBuild. I updated the postBuild version to 1.7.29 , as mentioned here https://methodshub.gesis.org/snippet/postBuild/ 

I fixed postBuild in other contents too, since they were using an older version.
